.github/workflows/test-misc.yml

```
@@ -73,6 +73,22 @@ jobs:
       - run: yarn message-system-sign-config
       - run: yarn test:unit
 
+  test-unit-windows:
+    description: "Uew windows runner to test platform specific utils"
```

actionlint (1.7.4)

unexpected key "description" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)